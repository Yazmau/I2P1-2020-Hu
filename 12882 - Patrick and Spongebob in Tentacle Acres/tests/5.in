1 72
7 88 30 33 71 16 69 72 50 93 73 41 24 51 31 20 93 39 20 25 58 57 94 60 8 8 97 65 54 53 75 38 38 69 20 20 20 6 71 78 79 34 3 30 29 24 34 61 3 5 49 55 68 64 86 29 57 50 57 22 47 11 96 82 77 28 15 90 5 83 13 74
8
1 1 5 71
1 1 9 26
1 1 56 63
1 1 52 66
1 1 36 60
1 1 17 52
1 1 58 60
1 1 48 66
