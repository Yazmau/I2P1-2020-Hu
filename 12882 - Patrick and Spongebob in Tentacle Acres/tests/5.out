97
94
96
96
86
97
57
96
